src/libraries/System.Collections.Concurrent/tests/EtwTests.cs

```diff
@@ -20,15 +20,17 @@ public void TestEtw()
                 {
                     var events = new ConcurrentQueue<int>();
 
-                    const int AcquiringAllLocksEventId = 3;
                     Clear(events);
-                    listener.RunWithCallback(ev => events.Enqueue(ev.EventId), () =>
-                    {
-                        var cd = new ConcurrentDictionary<int, int>();
-                        cd.TryAdd(1, 1);
-                        cd.Clear();
-                    });
-                    Assert.True(events.Count(i => i == AcquiringAllLocksEventId) > 0);
+
+                    // No longer fired after we moved ConcurrentDictionary to coreLib.
```

Comment:
Could we remove this code, or replace it with an assertion of it not happening anymore?